FreifunkMemorandumofUnderstanding_it.md

```diff
@@ -0,0 +1,67 @@
+# Memorandum d'intesa di Freifunk
+
+Dopo 10 anni Freifunk è decisamente un successo, ben conosciuta, e diffusa. Tuttavia osserviamo che le idee e gli obbiettivi originali di Freifunk non vengono presi in considerazione da tutte le community o da tutti i membri della community. Non è nostra intenzione escludere le comunità che non si identificano con la seguente interpretazione della natura di Freifunk. Ma è nostro interesse intraprendere un discorso di fondo su ciò che facciamo sotto l'etichetta condivisa di Freifunk. Questa bozza è stata scritta da un gruppo associato a Förderverein Freie Netzwerke e.V. L'idea è che costituisca un'intesa di fondo, e intende incoraggiare tutte le community a riflettere sugli argomenti trattati. Perché prendiamo parte a Freifunk? Quali idee ci sono dietro? Quali obbiettivi inseguiamo? In base a quali principi prendiamo decisioni e agiamo? Lo scopo di questo testo è sviluppare un'intesa che unisca tutti gli individui e i gruppi che agiscono sotto l'etichetta Freifunk.
+
+## Preambolo
+
+Le reti (wireless) libere sono costruite e messe a disposizione da una moltitudine di iniziative locali. Gli utenti sono al contempo operatori di reti di computer. Essi creano reti "fai da te", connettendo con le proprie forze appartamenti, case, vie, quartieri, paesi, o intere città. Una rete Freifunk è costruita in maniera decentralizzata e viene gestita da molti individui. Questa struttura organizzativa decentralizzata favorisce deliberatamente la volontà di attivarsi in loco invece di farsi governare da una entità più in alto. "Frei" (libero) sta a significare che queste reti sono fatte per essere aperte, accessibili anonimamente, gestite in ottica non commerciale, e le informazioni che in essa viaggiano non devono essere ispezionate, modificate o censurate. Sebbene Freifunk si riferisca principalmente a reti WLAN, il termine "freie Netzwerke" (reti libere) dovrebbe essere inteso in senso più ampio. Tra le altre cose, aderisce allo spirito della definizione fornita da freenetworks.org.
```

Comment:
Cambierei nella frase
_"Frei" (libero) sta a significare che queste reti sono fatte per essere aperte, accessibili anonimamente, gestite in ottica non commerciale, e le informazioni che in essa viaggiano non devono essere ispezionate, modificate o censurate._
la parte relativa alle informazioni - piu che non devono - non possono: magari in "_e il trasferimento di informazioni è al sicuro da ispezioni, modifche o censure_". Che ne dici?